mne/viz/evoked.py

```diff
@@ -779,7 +794,8 @@ def _animate_evoked_topomap(evoked, ch_type='mag', times=None, frame_rate=None,
 
 def plot_evoked_image(evoked, picks=None, exclude='bads', unit=True, show=True,
                       clim=None, xlim='tight', proj=False, units=None,
-                      scalings=None, titles=None, axes=None, cmap='RdBu_r'):
+                      scalings=None, titles=None, axes=None, cmap='RdBu_r',
+                      mask=None, alpha=.33, colorbar=True):
```

Comment:
Same here, set `alpha` to .5?

Reply:
I think 1/3 actually looks better .. but it's open for discussion.

Reply:
Better = easier to see what is masked.

Reply:
0.33 is fine with me, but then change it everywhere.